packages/internal-test-utils/shouldIgnoreConsoleError.js

```diff
@@ -3,6 +3,10 @@
 module.exports = function shouldIgnoreConsoleError(format, args) {
   if (__DEV__) {
     if (typeof format === 'string') {
+      if (format.startsWith('%c%s')) {
+        // Looks like a badged error message
+        args.splice(0, 3);
+      }
```

Comment:
This is ensuring we're filtering the same as before. No change in what gets filtered.